src/main/java/uk/gov/hmcts/reform/sscs/transform/deserialize/SubmitYourAppealToCcdCaseDataDeserializer.java

```diff
@@ -306,14 +309,18 @@ private static Subscription getRepresentativeSubscription(SyaCaseWrapper syaCase
 
             boolean emailAddressExists = StringUtils
                     .isNotBlank(syaCaseWrapper.getRepresentative().getContactDetails().getEmailAddress());
-            boolean phoneNumberExists =  StringUtils
-                    .isNotBlank(syaCaseWrapper.getRepresentative().getContactDetails().getPhoneNumber());
+            boolean isMobileNumber = PhoneNumbersUtil.isValidMobileNumber(
+                    syaCaseWrapper.getRepresentative().getContactDetails().getPhoneNumber());
+
+            final String mobileNumber = getPhoneNumberWithOutSpaces(syaCaseWrapper
+                    .getRepresentative().getContactDetails().getPhoneNumber());
```

Comment:
Is the representative guaranteed to exist at this point?
Could be worth doing a null check if not.

Reply:
Cheers!